nodejs/index.js

```diff
@@ -32,7 +32,7 @@ app.post('/messages', function(req, res) {
     const authorType = req.body.events[0].payload.message.author.type;
     if(authorType === 'user'){
         const conversationId = req.body.events[0].payload.conversation.id;
-        await sendMessage(appId, conversationId);
+        sendMessage(appId, conversationId);
```

Comment:
Is it because the `sendMessage` function is not marked as async, or because we just don't care about `await`ing the response?

Reply:
because this call itself isn't in a async function.

Reply:
🤦 :+1: